src/app/form-example/formExample.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import { FormDataService } from '../services/form-data.service';
+import { FormElementBase } from '../shared/dynamic-form/types/form-element-base';
+
+@Component({
+  selector: 'app-forms',
+  templateUrl: './formExample.component.html',
+  styleUrls: ['./formExample.component.scss'],
+  providers: [FormDataService],
+})
+export class FormExampleComponent {
+  myFormElements: FormElementBase<any>[] = [];
+
+  constructor(service: FormDataService) {
+    service.getFormData().subscribe({
```

Comment:
don't forget to unsubscribe on destroy

Reply:
have a look at this: https://medium.com/angular-in-depth/the-best-way-to-unsubscribe-rxjs-observable-in-the-angular-applications-d8f9aa42f6a0

Reply:
Unsubscribed on destroy.